README.md

```diff
@@ -24,6 +24,13 @@ Add the package to `app/AppKernel.php`, *before* the EzPublishCoreBundle declara
 
 The package will replace the services from the kernel, thus enabling the new features, such as multi-tagging.
 
+The application cache class needs to be customized. You can do so by setting the `SYMFONY_HTTP_CACHE_CLASS`
+environment variable:
+
+    export SYMFONY_HTTP_CACHE_CLASS='EzSystems\PlatformHttpCacheBundle\AppCache'
+
+Do not forget to restart your web server.
```

Comment:
"and set SYMFONY_HTTP_CACHE_CLASS accordingly."

if you extend class from this package you don't need to set the env variable, as you'll use the default AppCache class _(which will be the one to extend the class in this package)_.